scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -207,11 +207,17 @@ namespace <%= namespace %> {
         /// Determine whether the user can be shown to setup their native payment solution
         /// </summary>
         /// <returns>True if the user can be shown a native payment setup</returns>
```

Comment:
Need to update the docstring to mention the new call back parameter and void return.